design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -0,0 +1,212 @@
+type Mutation {
+    createNegotiableQuote(
+        cartId: ID! @doc(description: "Cart ID")
+        quoteName: String @doc(description: "Quote name")
+        comment: String @doc(description: "Comment")
+        files: [FileInput!] @doc(description: "Attached files")
+    ): CreateNegotiableQuoteOutput! @doc(description:"Generates negotiable quote request")
+
+    addNegotiableQuoteComments(
+        cartId: ID! @doc(description: "Cart ID")
+        comment: String @doc(description: "Comment")
+        files: [FileInput!] @doc(description: "Attached files")
+    ): AddNegotiableQuoteCommentsOutput! @doc(description:"Generates negotiable quote request")
+
+    updateNegotiableQuoteItemsQuantity(
+        cart_id: ID!, @doc(description: "unique Id of negotiable quote")
+        items: [ID!]! @doc(description: "unique Ids of Items to be update in negotiable quote")
```

Comment:
The `quantity` should be supplied for each item, not once for all items.

Consider the case where I'm updating the quantity of 3 items, and each has a separate quantity provided by the buyer.